assignment_2_Q2.py

```diff
@@ -0,0 +1,74 @@
+class Node:

```

Comment:
Can you try to group the common classes in a separate library, or use assignment_2_Q1.py as a library for assignment_1_Q2.py, as you prefer? In this way you ensure that your code is what is called a "single point of truth". For example, let's say you find a bug in the BST class while you are coding the Q2 file. You must remember afterwards to replicate the fix in the Q1 file. If you avoid duplication of the code it's easier, there is only one point to change.